docker-compose.yml

```diff
@@ -191,6 +200,12 @@ services:
       TPS_PENDING_POLLING_INTERVAL: ${TPS_PENDING_POLLING_INTERVAL}
       TPS_ERROR_POLLING_INTERVAL: ${TPS_ERROR_POLLING_INTERVAL}
       TPS_MONITORING_POLLING_INTERVAL: ${TPS_MONITORING_POLLING_INTERVAL}
+      CFS_PRIVATE_KEY: ${CFS_PRIVATE_KEY}
```

Comment:
Please update the scheduler readme file with the latest env variables

Reply:
updated